tests/layer_tests/tensorflow_tests/test_tf_Conv2DBackprop.py

```diff
@@ -0,0 +1,77 @@
+    # use_new_frontend - common parameter
+    def create_conv2dbackprop_placeholder_const_net(self, input_shape, input_filter, out_backprop, input_strides, input_padding, dilations, ir_version, use_new_frontend):
+        """
+            Tensorflow net                               IR net
+
+            Placeholder->Conv2DBackpropInput    =>       Placeholder->Transpose->ConvolutionBackpropData->Transpose
+                         /                                                       /
+            Placeholder-/                                Placeholder->Transpose-/
+
+        """
+
+        import tensorflow as tf
+
+        if dilations is None:
+            dilations = [1, 1, 1, 1] #default value regarding Documentation
+        else:
+            pytest.skip('Dilations != 1 isn\' supported on CPU by Tensorflow')
+
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            tf_input = tf.constant(input_shape)
+            tf_filter = tf.compat.v1.placeholder(tf.float32, input_filter, "Input")
+            tf_backprop = tf.compat.v1.placeholder(tf.float32, out_backprop, "InputBack")
+
+            tf.raw_ops.Conv2DBackpropInput(input_sizes = tf_input, filter = tf_filter, out_backprop = tf_backprop, strides = input_strides, padding = input_padding, dilations = dilations)
+
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        ref_net = None
+
+        return tf_net, ref_net
+
+    test_data = [
+        dict(input_shape=[1, 10, 10, 1], input_filter=[1, 1, 1, 1], out_backprop=[1, 10, 10, 1], input_strides=[1, 1, 1, 1], input_padding='SAME', dilations=None),
+        dict(input_shape=[1, 10, 10, 3], input_filter=[2, 2, 3, 3], out_backprop=[1, 10, 10, 3], input_strides=[1, 1, 1, 1], input_padding='SAME', dilations=None),
+        dict(input_shape=[1, 10, 10, 3], input_filter=[2, 2, 3, 3], out_backprop=[1, 5, 5, 3], input_strides=[1, 2, 2, 1], input_padding='SAME', dilations=None),
+        dict(input_shape=[1, 10, 10, 1], input_filter=[1, 1, 1, 1], out_backprop=[1, 10, 10, 1], input_strides=[1, 1, 1, 1], input_padding='VALID', dilations=None),
+        dict(input_shape=[1, 10, 10, 3], input_filter=[2, 2, 3, 3], out_backprop=[1, 9, 9, 3], input_strides=[1, 1, 1, 1], input_padding='VALID', dilations=None),
+        dict(input_shape=[1, 10, 10, 3], input_filter=[2, 2, 3, 3], out_backprop=[1, 5, 5, 3], input_strides=[1, 2, 2, 1], input_padding='VALID', dilations=None),
+        pytest.param(
+            dict(input_shape=[1, 56, 56, 3], input_filter=[2, 3, 3, 3], out_backprop=[1, 28, 28, 3], input_strides=[1, 2, 2, 1], input_padding='SAME', dilations=None),    
+            marks=pytest.mark.precommit_tf_fe),
+        pytest.param(
+            dict(input_shape=[1, 56, 56, 3], input_filter=[2, 3, 3, 3], out_backprop=[1, 28, 27, 3], input_strides=[1, 2, 2, 1], input_padding='VALID', dilations=None),    
```

Comment:
please try to use different filter shapes in these two samples

Reply:
done